lib/internal/http2/core.js

```diff
@@ -1657,13 +1658,15 @@ class Http2Stream extends Duplex {
                 'bug in Node.js');
   }
 
-  _write(data, encoding, cb) {
+  [kWriteGeneric](writev, data, encoding, cb) {
     // When the Http2Stream is first created, it is corked until the
     // handle and the stream ID is assigned. However, if the user calls
     // uncork() before that happens, the Duplex will attempt to pass
     // writes through. Those need to be queued up here.
     if (this.pending) {
-      this.once('ready', this._write.bind(this, data, encoding, cb));
+      this.once('ready', () => {
```

Comment:
There's no reason to change this line from `bind` to an arrow function.

Reply:
I just did that in order to make it a tad more consistent with the `net.js` equivalent:

```js
this.once('connect', () => {
      this._writeGeneric(writev, data, encoding, cb);
});
```

That said, perhaps the `.bind()` call is actually useful and I'm missing something, but I don't think the value of `this` will change, right? Why would we still bind it explicitly?

Reply:
@ryzokuken My guess would be that `.bind()` is slightly cheaper. Semantically, I think they should be equivalent.

Reply:
I would personally just not make any changes that aren't immediately necessary. IMO this unification can happen in the next PR, once `_writeGeneric` actually becomes shared between the two classes.